tgui/packages/tgui/interfaces/PaiInterface/Installed.tsx

```diff
@@ -91,22 +83,22 @@ const SoftwareButtons = (props, context) => {
           <Button
             disabled={door_jack}
             icon="plug"
-            onClick={() => act(currentSelection, { mode: DOOR_JACK.Cable })}
+            onClick={() => act(currentSelection, { mode: 'Cable' })}
```

Comment:
Ex: This used to have typo resilience, now it doesn't

Reply:
Kinda agree here, I was wondering if I should've constrained mode with a type, but felt (since my code anyway) that I was just being excessive